win32ss/gdi/gdi32/objects/font.c

```diff
@@ -2062,32 +2062,44 @@ RemoveFontResourceA(LPCSTR lpFileName)
 }
 
 /*
- * @unimplemented
+ * @implemented
  */
 BOOL
 WINAPI
 RemoveFontResourceExA(LPCSTR lpFileName,
                       DWORD fl,
-                      PVOID pdv
-                     )
+                      PVOID pdv)
 {
     NTSTATUS Status;
     LPWSTR lpFileNameW;
+    BOOL result;
 
-    /* FIXME the flags */
-    /* FIXME the pdv */
-    /* FIXME NtGdiRemoveFontResource handle flags and pdv */
+    if (fl & ~(FR_PRIVATE | FR_NOT_ENUM))
+    {
+        SetLastError(ERROR_INVALID_PARAMETER);
+        return FALSE;
+    }
 
-    Status = HEAP_strdupA2W ( &lpFileNameW, lpFileName );
-    if (!NT_SUCCESS (Status))
-        SetLastError (RtlNtStatusToDosError(Status));
-    else
+    _SEH2_TRY
+    {
+        Status = HEAP_strdupA2W(&lpFileNameW, lpFileName);
```

Comment:
Existing calls do not have SEH. Why is it needed here?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=HEAP_strdupA2W

Reply:
I can confirm SEH handling in this function of Windows.

![image](https://github.com/user-attachments/assets/3a82522a-ef5d-4a56-bcd4-1b4c39f36f9a)